_posts/2023-3-13-collaborative-trailheads.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Trailheads, a Collaborative Approach"
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2023-03-13 12:00:00
+categories:
+  - Featured
+tags:
+  - data
+  - trailheads
+  - OSM
+  - OpenStreetMap
+---
+
+At UGRC, we serve up datasets from a vast array of sources. Some, like municipal boundaries, are created in-house. Others are pulled from a single steward, like the land ownership data maintained by SITLA. A large part of our work is aggregating datasets from multiple stewards into coherent, state-wide datasets. Roads, parcels, and address points all fall into this category. 
+
+However, there are some datasets that are so broad in their geographic scope and so dynamic in their content that even an aggregation strategy fails to keep up with their changes and fails to capture all the possible data sources.
+
+Trailheads are great examples of this. They receive frequent updates and are governed by a variety of entities including cities and towns, counties, land preservation trusts, BLM, USFS, State Parks, and National Parks, to name a few. Because of this, it can be a challenge to keep this layer current in the SGID. But, what if we used a collaborative approach to manage this data?
+
+
+### OpenStreetMap and Collective Stewardship
+{: .flex .flex--left}
+
+At UGRC, we see [OpenStreetMap](https://www.openstreetmap.org/) (OSM) as an ideal platform for managing and maintaining datasets such as trails, trailheads, campgrounds, and a variety of other layers - because who better to steward this data than all of us? 
+
+OpenStreetMap (OSM) is a collaborative platform for collectively editing data in real-time. In the case of trailheads, mapping enthusiasts can work side-by-side (so to speak) along the data stewards to keep the data up-to-date.  
+
+We already have an [established OSM workflow]({% link _posts/2022-03-21-introducing-open-source-places.md %}) for the [Open Source Places SGID layer]({% link data/society/open-source-places/index.html %}). This workflow allows us to source the data directly from OSM. Our goal is to eventually incorporate this collective stewardship model into as many suitable datasets as we can. For now, we have identified [trailheads]({% link data/recreation/trails/index.html %}) as the next suitable SGID layer for this collaborative editing model.
+
+### Collaborate Trailheads
```

Comment:
Should this be "Collaborative"? It sounds like an imperative right now (Collaborate, Trailheads!  ... Hmm, is a trailhead a person who's always on the trails? Like a deadhead?)

```suggestion
### Collaborative Trailheads
```

Reply:
Otherwise, I like it! I'm a fan of having an image or two to break up the text, but that takes some time to implement and I know Matt is anxious to get the the newsletter out.